src/main/scala/codacy/checkstyle/DocGenerator.scala

```diff
@@ -59,13 +59,14 @@ object DocGenerator {
               .drop(1)
               .map(_.\\("td").to[List])
               .collect {
-                case name :: description :: _ :: default :: _ =>
+                case name :: description :: tpe :: default :: _ =>
                   val defaultValue = Option({
                     // Remove spaces and breaklines in default values
                     val defVal = default.text.replaceAll("""\n\s+""", "").trim.stripSuffix(".")
-                    // Remove quotes around default values
-                    if (defVal.startsWith("\"") && defVal.endsWith("\"")) {
-                      defVal.stripPrefix("\"").stripSuffix("\"")
+                    // Remove quotes around regular expressions
+                    if (tpe.text.trim == "Regular Expression" && defVal != "null") {
+                      // Leaves only what's inside outer quotes.
+                      defVal.dropWhileLeftRight(_ != '"').drop(1).dropRight(1)
```

Comment:
this implementation can easily fail, `drop` and `dropRight` are going to throw exceptions instead of avoiding to strip ( https://www.scala-lang.org/api/2.12.4/scala/collection/immutable/StringOps.html#stripSuffix(suffix:String):String )